test/erlfmt_format_SUITE.erl

```diff
@@ -2574,13 +2574,31 @@ receive_expression(Config) when is_list(Config) ->
         "    2 -> two\n"
         "end\n"
     ),
-    % ?assertSame(
-    %     "receive\n"
-    %     "after\n"
-    %     "    % foo\n"
-    %     "    0 -> ok\n"
-    %     "end\n"
-    % ),
+    ?assertFormat(
+        "receive\n"
+        "after\n"
+        "    0 -> ok\n"
+        "    % receive post comment\n"
+        "end\n",
+        "receive\n"
+        "after 0 ->\n"
+        "    ok\n"
+        "    % receive post comment\n"
+        "end\n"
+    ),
+    ?assertFormat(
+        "receive\n"
+        "after\n"
+        "    % before zero\n"
+        "    0 -> ok\n"
+        "end\n",
+        "receive\n"
+        "after\n"
+        "    % before zero\n"
+        "    0 ->\n"
+        "        ok\n"
```

Comment:
Do we force this to break if there's a comment? I don't think we should

Reply:
No longer forced